app/config/services.php

```diff
@@ -199,4 +199,17 @@
         'optional' => false,
         'icon' => '',
     ],
+    'assistant' => [
+        'key' => 'assistant',
+        'name' => 'Assistant',
+        'subtitle' => 'The Appwrite assistant service allows you to interact with appwrite assistant.',
```

Comment:
```suggestion
        'subtitle' => 'The Appwrite assistant service allows you to interact with Appwrite'\s AI assistant.',
```

Reply:
Removing this for now, adding it to console service